addons/survey/static/src/js/survey_form.js

```diff
@@ -1029,6 +1044,28 @@ publicWidget.registry.SurveyFormWidget = publicWidget.Widget.extend({
         return false;
     },
 
+   /**
+    * Load the target section background and render it when loaded.
+    * This method is used to pre-load the image during the questions transitions (fade out) in order to be sure the
+    * image is fully loaded when setting it as background of the next question and finally display it (fade in)
+    *
+    * @param {string} imageUrl
+    * @private
+    */
+    _preloadBackground: async function (imageUrl) {
+        var resolvePreload;
+        var preloadPromise = new Promise(function (resolve, reject) {resolvePreload = resolve;});
+
+        // Wait until new background is loaded before changing the background
+        var background = new Image();
```

Comment:
// We have to manually create a promise here because the "onload" API does not provide one.